examples/sudoku/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2016 Satyaki Upadhyay
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+
+add_hpx_component(sudoku
+  SOURCE_GLOB "sudoku.cp*"
+  HEADER_GLOB "sudoku.hp*"
```

Comment:
Any particular reasons for the use of globs here, or is it inspired by an example?